### PR TITLE
fix: add missing system configuration fileds in swagger

### PR DESCRIPTION
**Pull request by chlins**
Signed-off-by: chlins <chlins.zhang@gmail.com>

### CODE REVIEW

File: api/v2.0/legacy_swagger.yaml [hunk 1]
```diff
@@ -3309,6 +3312,21 @@ definitions:
...
+        type: string
+        description: The endpoint of uaa server.
+      uaa_client_id:
+        type: string
+        description: The client id of uaa.
+      uaa_client_secret:
+        type: string
+        description: The client secret of uaa.
+      uaa_verify_cert:
+        type: string
```
Comment (stonezdj) [inline api/v2.0/legacy_swagger.yaml:3328]: uaa_verify_cert is  boolean

Comment (chlins) [reply api/v2.0/legacy_swagger.yaml]: updated.

**Comment by chlins** [comment]
@stonezdj Can you take a look again? Thanks.

File: api/v2.0/legacy_swagger.yaml [hunk 2]
```diff
@@ -3459,7 +3497,11 @@ definitions:
         description: The default storage quota for the new created projects.
       token_expiration:
         $ref: '#/definitions/IntegerConfigItem'
-        description: 'The expiration time of the token for internal Registry, in minutes.'
+        description: 'The expiration time of the token for internal Registry, in
+        minutes.'
+      robot_token_duration:
+        $ref: '#/definitions/IntegerConfigItem'
+        description: 'The expiration time of the robot account, in minutes.'
```
Comment (heww) [inline api/v2.0/legacy_swagger.yaml:3504]: Please change `in minutes` to`in days` in harbor 2.2

Comment (chlins) [reply api/v2.0/legacy_swagger.yaml]: updated.

File: api/v2.0/legacy_swagger.yaml [hunk 3]
```diff
@@ -3344,7 +3362,11 @@ definitions:
         description: The default storage quota for the new created projects.
       token_expiration:
         type: integer
-        description: 'The expiration time of the token for internal Registry, in minutes.'
+        description: 'The expiration time of the token for internal Registry, in
+        minutes.'
```
Comment (heww) [inline api/v2.0/legacy_swagger.yaml:3366]: Join them to one line?

File: api/v2.0/legacy_swagger.yaml [hunk 4]
```diff
@@ -3360,7 +3382,11 @@ definitions:
               daily_time:
                 type: integer
                 description: 'The offset in seconds of UTC 0 o''clock, only valid when the policy type is "daily"'
-            description: 'The parameters of the policy, the values are dependant on the type of the policy.'
+            description: 'The parameters of the policy, the values are dependant
+            on the type of the policy.'
```
Comment (heww) [inline api/v2.0/legacy_swagger.yaml:3386]: Join them to one line?

**Review by heww — APPROVED** [review]

**Comment by chlins** [comment]
@stonezdj @reasonerjt Please take a look, thanks.

**Comment by stonezdj** [comment]
Because the master branch already contains the change, this PR can be closed

**Pull request closed without merging**